src/main/java/com/amazon/opendistroforelasticsearch/security/dlic/rest/api/AbstractApiAction.java

```diff
@@ -555,4 +557,12 @@ protected boolean isReservedAndAccessible(final SecurityDynamicConfiguration<?>
 		return true;
 	}
 
+	protected boolean isHiddenAndAccessible(final SecurityDynamicConfiguration<?> existingConfiguration,
```

Comment:
Is it necessary to introduce this method? Should `isHidden()` be modified to exclude super admin?

Reply:
I was thinking about modifying but there are some occurrences in the code where they require isHidden and hence can not modify.
Also it is consistent to the isReserved, but thats a second excuse.

Reply:
The PR makes "hidden" resource to be "visible" to super admin, should not the other occurrences of `isHidden()` be replaced too?

Reply:
Most of them are replaced.
The only one which is not replaced is the GET request where we have to specify if it is a hidden resource is not in the response 

https://github.com/opendistro-for-elasticsearch/security/blob/master/src/main/java/com/amazon/opendistroforelasticsearch/security/dlic/rest/api/AccountApiAction.java#L140

Reply:
IMO, it will be better to change one place instead of changing multiple locations. Introduce `isHidden(String resourceName)` into `SecurityDynamicConfiguration` and call it directly from GET.

Reply:
I can remove the isHidden function alltogether itself from AbstractApiAction as it might look like some code duplication. 
isHidden will point to SecurityDynamicConfiguration and wherever supderAdmin check is required will be added like 
`isHidden && superAdmin()`

Or do you want the way you said. IsHidden at both the location, only single GET request points to `SecurityDynamicConfiguration`